splunk_event/test_splunk_event.py

```diff
@@ -184,22 +184,24 @@ def test_not_dispatch_sids_checks(self, mocked_splunk_helper):
         second_persistent_data = self.check.status.data.get(instance.get('url') + "minimal_events")
         # The second run_check will finalize the previous saved search ids and create a new one,
         # so we make sure this is the case
-        self.assertNotEqual(first_persistent_data[0][1], second_persistent_data[0][1])
+        self.assertEqual(first_persistent_data, second_persistent_data)
```

Comment:
why they are now the same ?

Reply:
because sid is the same for the same saved search and previously I was storing the saved search object along with sid which used to be different everytime.